shared/src/native-src/string.cpp

```diff
@@ -152,6 +153,18 @@ namespace shared {
         return str.size() >= prefix.size() && str.compare(0, prefix.size(), prefix) == 0;
     }
 
+    WSTRING to_lower(const WSTRING& str)
+    {
+        WSTRING lower(str);
+        transform(lower.begin(), lower.end(), lower.begin(), ::towlower);
+        return lower;
+    }
+
+    bool string_lower_equal(WSTRING const& s1, WSTRING const& s2)
+    {
+        return s1 == s2;
+    }
```

Comment:
do we need `string_lower_equal()` anymore? seems like you can just to the `==` directly now.

Reply:
🤦 
Okay yes we can just do that thanks!

Reply:
Removed

Reply:
unresolving thread because I still see `string_lower_equal()` 😅

Reply:
Okay I think I got it now 🙇